float32[7] jointPos